idseq_pipeline/commands/common.py

```diff
@@ -229,6 +229,21 @@ def percent_str(percent):
     except:
         return str(percent)
 
+def extract_m8_readid(read_id_column):
+    # Get raw read ID without our annotations:
+    # If m8 is from gsnap (case 1), 1 field has been prepended (taxid field).
+    # If m8 is from rapsearch in an old version of the pipeline (case 2), 3 fields have been prepended (taxid, 'NT', NT accession ID).
+    # If m8 is from rapsearch in a newer version of the pipeline (case 3), many fields have been prepended (taxid, alignment info fields),
+    # but the delimiter ":read_id:" marks the beginning of the raw read ID.
```

Comment:
maybe we should remove case 3 here as it didn't end up existing?

Reply:
will it be reinstated?

Reply:
I don't think so, because I introduced it in a stale PR that tried to do the same alignment stuff you are doing here :)

Reply:
Will close now.

Reply:
As yours works much better ;)

Reply:
But I do like that format. The current format is kinda like guess work.

Reply:
You could always introduce that format here then! Thus making the documentation true retroactively